src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/ResourceManagerRpc.java

```diff
@@ -124,5 +125,15 @@ Integer getInt(Map<Option, ?> options) {
    */
   List<Boolean> testPermissions(String projectId, List<String> permissions);
 
+  /**
+   * Gets the latest state of a long-running operation.
+   *
+   * <p>Clients can use this method to poll the operation result at intervals as recommended by the
+   * API service.
+   *
+   * @param name The name of the operation resource.
+   * @return
```

Comment:
If you have `@param` then it is advisable to have `@return`.

Reply:
done